rflx/pyrflx/typevalue.py

```diff
@@ -761,13 +761,13 @@ def set_field_without_size(
             first = self._get_first(field_name)
```

Comment:
For each bug fix, a regression test should be added, so that the issue cannot be accidentally introduced again.